LayoutTests/fast/harness/test-wait.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html class="test-wait">
+<body>
+<div id='result'>FAIL</div>
+<script>
+if (window.testRunner)
+    testRunner.dumpAsText();
+
+onload = () => {
+    setTimeout(() => {
+        result.textContent = 'PASS';
+        document.documentElement.removeAttribute('class');
+    }, 100);
```

Comment:
Does this have to be as long as 100ms?. That's a pretty slow test.

Reply:
You could do something like have two requestAnimationFrames, remove `test-wait` in the first one, and make sure the second one's log doesn't show in the results.

Reply:
oh, I was just using 100ms as a safe time. I reduced it to 10ms.